instrumented-tests/http-server-mock/python/start_mock_server.py

```diff
@@ -47,6 +47,14 @@ def do_GET(self):
             (r"/inspect$", self.__GET_inspect),
         ])
 
+    def do_DELETE(self):
+        """
+        Routes all incoming DELETE requests
+        """
+        self.__route([
+            (r"/requests$", self.__DELETE_requests),
+        ])
+
```

Comment:
I'm adding this endpoint to remove recorded requests so we can clear the mock server before restarting the core instance.